### DISCUSSION TITLE
Calls to details.php fail with Failed to parse time string (@) at position 0 (@): Unexpected character

### DISCUSSION THREAD

**Discussion by 942u3895hjf** [osm-search/Nominatim#2400] [Q&A]
Hello,

I just installed 3.7.2 on Ubuntu 20.04 and imported some OSM files i recently got from Overpass exports and a file from Geofabrik, the Australia.osm. It took a little while but it went fine.

Calls to details.php fail, such as:
http://localhost/nominatim/details.php?osmtype=W&osmid=6397441&addressdetails=1&hierarchy=0&group_hierarchy=1&format=json

Giving:
{"error":{"code":0,"message":"DateTime::__construct(): Failed to parse time string (@) at position 0 (@): Unexpected character"}}

I wouldexpect is the output as seen here:
https://nominatim.openstreetmap.org/details.php?osmtype=W&osmid=6397441&addressdetails=1&hierarchy=0&group_hierarchy=1&format=json

I do get errors when calling: ./utils/check_import_finished.php
PHP Warning: require(Symfony/Component/Dotenv/autoload.php): failed to open stream: No such file or directory in /srv/nominatim/Nominatim-3.7.2/lib-php/dotenv_loader.php on line 3
PHP Fatal error: require(): Failed opening required 'Symfony/Component/Dotenv/autoload.php' (include_path='.:/usr/share/php') in /srv/nominatim/Nominatim-3.7.2/lib-php/dotenv_loader.php on line 3

I must be doing something wrong, any ideas?

Thanks!

**Comment by lonvia** [comment]
You should use the `nominatim` tool for all operations in 3.7.2. So: `nominatim admin --check-database`

Regarding the date error, run the same URL with `&debug=1` and check what the output is there.
